airflow_tecton/operators/job_utils.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2022 Tecton, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import pprint
+import time
+
+
+def is_job_running(hook, workspace, feature_view, online, offline, allow_overwrite=False, start_time=None, end_time=None, job_type="batch"):
```

Comment:
this seems to actually return the job, so the name is also confusing

Reply:
since this job is not needed for feature table, going to revert this function.